fedelemflowlist/input/FlowExclusions.csv

```diff
@@ -0,0 +1,45 @@
+Flowable,Context
+"Water, fresh",resource/water/brackish water body
+"Water, fresh",resource/water/brackish water body/lake
+"Water, fresh",resource/water/brackish water body/lake/rural
+"Water, fresh",resource/water/brackish water body/lake/urban
```

Comment:
is there a brackish water body/river option?

Reply:
no